src/actions/posts.js

```diff
@@ -623,12 +623,12 @@ export function getPosts(channelId, page = 0, perPage = Posts.POST_CHUNK_SIZE) {
     };
 }
 
-export function getPostsUnread(channelId) {
+export function getPostsUnread(channelId, fetchRoot = true) {
     return async (dispatch, getState) => {
         const userId = getCurrentUserId(getState());
         let posts;
         try {
-            posts = await Client4.getPostsUnread(channelId, userId);
+            posts = await Client4.getPostsUnread(channelId, userId, 30, 30, fetchRoot);
```

Comment:
Can we change the 30, 30 with constants?

Reply:
will do